frontend/src/components/AttendeeExpansionPanel/LmAttendeeAddForm.vue

```diff
@@ -229,7 +232,9 @@ const requiredRule = [
         </v-btn>
       </v-defaults-provider>
       <v-defaults-provider :defaults="{ VIcon: { color: '#fff' } }">
-        <v-btn style="flex: 1" color="primary" prepend-icon="mdi-check" variant="flat" type="submit"> Speichern </v-btn>
+        <v-btn style="flex: 1" color="primary" :loading="loading" prepend-icon="mdi-check" variant="flat" type="submit">
```

Comment:
Loading only works on edit not on create

Reply:
Fixed :)